pandas/tools/merge.py

```diff
@@ -705,7 +705,7 @@ def _get_join_info(self):
                 _left_join_on_index(left_ax, right_ax, self.left_join_keys,
                                     sort=self.sort)
 
-        elif self.left_index and self.how == 'right':
+        elif self.tleft_index and self.how == 'right':
```

Comment:
pls don't change code in the same PR as docs (unless its germane), I assume this was a typo though.